samples/http-client-examples/server.mts

```diff
@@ -36,6 +36,10 @@ app.get("/origin", async (req, res) => {
   res.send(JSON.stringify(result));
 });
 
+app.get("/destination", async (req, res) => {
```

Comment:
[pp] perhaps rename it to a localDestinationTestEndpoint so that it is clear that this does not give you a destination.